services/models/namespace.go

```diff
@@ -6,7 +6,27 @@ import (
 	"github.com/kiali/kiali/kubernetes"
 )
 
+// A Namespace is the main classification in kiali.
```

Comment:
I guess we should align the terms with kubernetes, istio and/or openshift.
As we use the "Namespace" concept in the same way as k8s, nothing wrong to re-use same wording from their API for same things (i.e. namespace).
https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.10/#namespace-v1-core